lib/log.js

```diff
@@ -0,0 +1,33 @@
+/*
+ Copyright Red Hat, Inc., and individual contributors
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+var DEBUG = false;
```

Comment:
Would you see any value in being able to set this without having to modify this file? Like have it optionally get it from an environment variable or something?